src/workers/jobs.js

```diff
@@ -254,15 +238,37 @@ export async function uploadContacts(job) {
     await CampaignContact.save(savePortion);
   }
 
-  const optOutCellCount = await r
+  const optOutCellResults = await r
     .knex("campaign_contact")
     .whereIn("cell", function optouts() {
       this.select("cell")
         .from("opt_out")
         .where("organization_id", campaign.organization_id);
     });
-  console.log('DEBUG: uploadContacts 3', campaignId, optOutCellCount);
+  const optOutCellCount = optOutCellResults.length;
```

Comment:
This seems a bit risky -- if optOutCellResults is undefined/null then this will error.
There's a better pattern for just getting the count anyway (for the final version):
[r.getCount](https://github.com/MoveOnOrg/Spoke/blob/main/docs/EXPLANATION-development-guidelines.md#db-queries)

Reply:
Thanks for the pointer! I've updated it to use `await r.getCount` instead.